### PR TITLE
186 open snapshot in editor

### PR DESCRIPTION
**Pull request by gregveres**
This PR does two things. It implements #186 and #191. 
For  #186, this creates a new button to load the snapshot in the editor so that you can quickly
see the snapshot - useful when writing tests. 

I had created FR #191 because I wanted a way to generate the snapshots, not realizing that the
button was already there. This change makes the button visible all the time. The reason for that
is that there are times when you know the snapshots are out of date and need to be updated.
If you run the tests just to have them fail, it wastes time. Having this button visible all the time
saves time because you can generate the snapshots when you know they are out of date.

### CODE REVIEW

**Comment by gregveres** [comment]
linked issue: #186

File: ui/test-file/summary/index.tsx [hunk 1]
```diff
@@ -188,9 +195,8 @@ export default function FileSummary({
             }}
           />
         </Tooltip>
-        {haveSnapshotFailures && (
```
Comment (Raathigesh) [inline ui/test-file/summary/index.tsx:191]: Don't want want to show this button only when there is a snapshot failure?

Comment (gregveres) [reply ui/test-file/summary/index.tsx:191]: Correct. I think the button to generate snapshots should always be available. Most times you know that you have made a change that will break the snapshots. It is a waste of time to have to run the tests to wait for the button to appear and then press it. I would prefer to have the option to generate the snapshots when I know they need generating.

Comment (Raathigesh) [reply ui/test-file/summary/index.tsx:191]: That's a good point.

**Comment by Raathigesh** [comment]
If you could help with the merge conflict, I can get this merged as well and release a new version by this week.

**Comment by gregveres** [comment]
@Raathigesh sorry, I don't know how those became merge conflicts and GitHub didn't suggest the proper merge for one of them. I have resolved the conflicts now with the code that should be there.

**Comment by Raathigesh** [comment]
Thanks for this as well. Looks good. 

I'm planning to a sanity test this weekend and will do a release with all your changes.

**Pull request merged**